torch/_subclasses/fake_tensor.py

```diff
@@ -26,6 +26,18 @@
 
 CONSTANT_NUMEL_LIMIT = 1
 
+CONSTANT_LIFTING = False
```

Comment:
this should be a property of the FakeTensorMode, not a global property. Maybe `convert_non_fake_inputs`.